changelog.md

```diff
@@ -1,3 +1,12 @@
+# 3.11.0
```

Comment:
shrinkwrap doesn't work currently with sequelize? If not i'd say this is a bug fix then, not a new feature.